client/my-sites/plugins/plugin-autoupdate-toggle/index.jsx

```diff
@@ -16,12 +16,17 @@ module.exports = React.createClass( {
 	displayName: 'PluginAutopdateToggle',
 
 	propTypes: {
+		isMock: React.PropTypes.bool,
 		site: React.PropTypes.object.isRequired,
 		plugin: React.PropTypes.object.isRequired,
 		wporg: React.PropTypes.bool
 	},
 
 	toggleAutoupdates: function() {
+		if( this.props.isMock ) {
```

Comment:
Fixed the `if(`. I agree that the toggles are important. They are pretty much all the point of this section ... I think being able to see that you could interact with your plugins from there would be the more important part of this preview.

About how to disable them without passing a prop... well, if we could do something like what you are saying, something in the parent component, what would be neat. But I can't really figure out how to do it in another way, so any idea would be more than welcome :)